sql/moz-fx-data-shared-prod/google_ads_derived/android_app_campaign_stats_v1/checks.sql

```diff
@@ -1,2 +1,2 @@
 #fail
-{{ is_unique(["date", "campaign", "ad_group"], "date = DATE_SUB(@submission_date, INTERVAL 27 DAY)") }}
+{{ is_unique(["date", "campaign", "ad_group"]) }}
```

Comment:
@kwindau considering the query is running with a 27 day lag, this specific filter that was removed here was actually required. Right now this will most likely target partitions that do not exist.